Sources/Testing/Testing.docc/Expectations.md

```diff
@@ -14,25 +14,40 @@ Check for expected values and outcomes in tests.
 
 ## Overview
 
-Tests may validate behaviors using expectations. This page describes the various
-built-in expectation APIs.
+The testing library provides `#expect()` and `#require()` macros you use to 
+validate expected outcomes and a `Confirmation` type you can use to confirm the 
+occurrence of asynchronous events. To validate that an error is thrown, or _not_ thrown, 
+the testing library provides several overloads of the macros that you can use.
 
 ## Topics
 
-### Validating behavior using expectations
+### Evaluating conditions
```

Comment:
"Evaluating" is not a word we generally use around expectations, I think? We use "check" in the implementation—perhaps "Checking expectations and requirements"? And lower down, "Checking thrown errors"?

Reply:
I talked to @stmontgomery and we don't want to say "requirements"—instead, they're "required expectations." "Requirement" could be confused with "protocol requirements."